recipes/tracy/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class TracyConan(ConanFile):
+    name = "tracy"
+    description = "C++ frame profiler"
+    topics = ("profiler", "performance", "gamedev")
+    homepage = "https://github.com/wolfpld/tracy"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ["BSD-3-Clause"]
+    settings = "os", "compiler", "build_type", "arch"
+
+    # Existing CMake tracy options with default value
+    tracy_options = {
```

Comment:
```suggestion
    _tracy_options = {
```
Any custom attribute or method must be protected.

Reply:
Fixed